Index: sites/all/modules/prototype/contrib/mcfilemanager/mcfilemanager.pages.inc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sites/all/modules/prototype/contrib/mcfilemanager/mcfilemanager.pages.inc	(revision e8d3f4ba98d84dfdefe14072118f9fd0a3e9c7e3)
+++ sites/all/modules/prototype/contrib/mcfilemanager/mcfilemanager.pages.inc	(revision )
@@ -119,7 +119,8 @@
   $library_path = mcfilemanager_library_directory();
 
   // Check if the file auth_drupal.php resides in Drupal's root directory
-  if (!file_exists('auth_drupal.php')) {
+  $module_path = drupal_get_path('module','mcfilemanager');
+  if (!file_exists("$module_path/auth_drupal.php")) {
     $requirements['mcfilemanager_auth_file'] = array(
       'title' => $t('MCFileManager authentication file'),
       'value' => $t('The required file %file could not be found! Refer to MCFileManager\'s !help section for installation instructions.', array('%file' => 'auth_drupal.php', '!help' => l(t('help'), 'admin/help/mcfilemanager'))),
